packages/ui/src/forum/components/threads/ThreadBrowser.tsx

```diff
@@ -46,6 +46,29 @@ export const ThreadBrowser = ({ label }: ThreadBrowserProps) => {
     setCurrentItemsGroup(currentItemsGroup + 1)
   }
 
+  if (!items) {
```

Comment:
There is a lot of duplicated code here, but I suppose that stems from not being sure how it's going to work with the query node. I think it can be simplified alongside coding in querying for these threads.